serverless.yml

```diff
@@ -0,0 +1,31 @@
+service: clear-lambda-storage
+
+provider:
+  name: aws
+  stage: production
+  runtime: python3.6
+  region: us-east-1
```

Comment:
Maybe add here support for opt? (So users will be able to define which region to deploy to).
Otherwise looks great! Tested on my end as well

Reply:
Ok! I've added 🤘🏼